apps/web/src/app/map.tsx

```
@@ -0,0 +1,17 @@
+import { LocationsWithUsers } from "@/db/locations";
+
+export interface MapProperties {
+  locations: LocationsWithUsers;
+}
+
+export default function Map({ locations }: MapProperties) {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)